### PR TITLE
Re-add type-bounded signatures of Views.extendValue and Views.expandValue

### PR DESCRIPTION
**Pull request by imagejan**
See [this discussion](https://gitter.im/imglib/imglib2?at=5e2abf31e177666195bd4689) on gitter.

### CODE REVIEW

File: src/main/java/net/imglib2/view/Views.java [hunk 1]
```diff
@@ -1356,6 +1375,24 @@ public static boolean isZeroMin( final Interval interval )
...
+	 * @param source
+	 *            the interval to expand.
+	 * @param t
+	 *            Constant extension of source.
+	 * @return Expansion of the {@link RandomAccessibleInterval} source as
+	 *         specified by t and border.
+	 * @deprecated use {@code expandValue} with unbounded type parameter T
+	 */
+	@Deprecated
+	public static < T extends RealType< T > > IntervalView< T > expandValue( final RandomAccessibleInterval< T > source, final T t, final long... border )
```
Comment (hanslovsky) [inline src/main/java/net/imglib2/view/Views.java]: This can be relaxed to `Type<T>` instead of `RealType<T>`, no?

Comment (tpietzsch) [reply src/main/java/net/imglib2/view/Views.java]: Yes, it should be as it was before #268

Comment (imagejan) [reply src/main/java/net/imglib2/view/Views.java]: Oops, you're right, I guess..

Comment (imagejan) [reply src/main/java/net/imglib2/view/Views.java]: Now changed and force-pushed.

**Pull request merged**